### PR TITLE
Added plugin to get new relic browser data from api

### PR DESCRIPTION
**Pull request by guilherme-carneiro-ca**
## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
We have task Scheduler to collect default metrics from prometheus and chart and gauge classes to collect metrics from this type of chart.

## What is the new behavior?
Removed task scheduler because we need to expose only one endpoint to collect the data. Prometheus will access this endpoint according to the stipulated time.

Added Gauge chart to collect a percentage of pageviews with javascript errors.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/plugins/newrelic_browser/api-request.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+var rp = require('request-promise');
```
Comment (cirocfc) [inline src/plugins/newrelic_browser/api-request.js]: const

File: src/plugins/newrelic_browser/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const jsErrors = require('./javascriptErrorsPercent/javascriptErrorsPercent')
+
+const collectData = function(){
```
Comment (cirocfc) [inline src/plugins/newrelic_browser/index.js]: () => { }

File: src/app/routes/metrics.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 const Register = require('../../plugins/prometheus').register;
+const newRelicPlugin = require('../../plugins').newRelicPlugin;
 
 const metrics = (req, res) => {
   res.set('Content-Type', Register.contentType);
-  res.end(Register.metrics());
+  newRelicPlugin.collectData().then((response) => {
```
Comment (cirocfc) [inline src/app/routes/metrics.js]: (response => {

File: package.json [hunk 4]
```diff
@@ -25,6 +25,7 @@
     "node-cron": "^2.0.1",
     "prom-client": "^11.1.3",
     "request": "^2.88.0",
```
Comment (cirocfc) [inline package.json:27]: It isn't necessary anymore, is it?

File: src/plugins/newrelic_browser/javascriptErrorsPercent/javascriptErrorsPercent.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        name: 'newrelic_browser_javascript_errors_percent',
+        help: 'return a percentage of pageviews with javascript errors',
+      });
+    }
+
+    const names = 'EndUser/errors';
+    const values = 'error_percentage';
+    browser
+      .collectData(names, values)
+      .then(function (response) {
```
Comment (cirocfc) [inline src/plugins/newrelic_browser/javascriptErrorsPercent/javascriptErrorsPercent.js]: () => {

File: package.json (hunk 4 above)
Comment (guilherme-carneiro-ca) [reply package.json:27]: It's necessary because request is defined as a peer-dependency.
https://github.com/request/request-promise#installation

Comment (cirocfc) [reply package.json:27]: :+1:

**Review by cirocfc — APPROVED** [review]

**Pull request merged**